extensions/amp-accordion/amp-accordion.md

```diff
@@ -3,7 +3,6 @@ $category@: layout
 formats:
   - websites
   - ads
```

Comment:
Would you mind also removing ads while you're here?

Reply:
Wasn't sure if ads will support it, thanks for confirming. Done.

Reply:
Thank you! I'll merge once the build completes.

Reply:
looks like it's completed now!